src/applications/letters/tests/actions/letters.unit.spec.js

```diff
@@ -26,10 +27,19 @@ import {
   getLetterPdf,
 } from '../../actions/letters';
 
-Sentry.init({
-  dsn: 'http://one@fake/dsn',
-  transport: sentryTransport,
-});
+/**
+ * commenting this out due to this error that occurs when running all tests
+ * after the JSDOM upgrade:
+ *
+ * node_modules/jsdom/lib/jsdom/living/helpers/create-event-accessor.js:9
+ * el.addEventListener(eventName, event => {
+ *    ^
+ * TypeError: el.addEventListener is not a function:
+ */
+// Sentry.init({
```

Comment:
This line was triggering the new error. Perhaps updating the Sentry deps would solve it? I'm not sure what's going on with this.